test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/DefaultTagHelperContentTest.cs

```diff
@@ -475,11 +475,11 @@ public void Append_WrittenAsEncoded()
         }
 
         [Fact]
-        public void AppendEncoded_DoesNotGetEncoded()
+        public void AppendHtml_DoesNotGetEncoded()
```

Comment:
Shocked we don't have a `SetHtmlContent` equivalent. Could you add a single test while you're here real quick?